modules/nodes-edge-remote-workers-strategies.adoc

```diff
@@ -0,0 +1,157 @@
+
+The control plane, by default, applies taints and evicts pods at a rate of 0.1 nodes/sec. However, in a cluster that uses Kubernetes zones, pod eviction behavior is altered.
+
+If a zone is fully disrupted, where all nodes in the zone have a `Ready` condition that is `False` or `Unknown`, the control plane does not apply the taint to the nodes in that zone.
+
+For partially disrupted zones, where more than 55% of the nodes have a `False` or `Unknown` condition, the pod eviction rate is reduced to 0.01 nodes/sec. Nodes in smaller clusters, with fewer than 50 nodes, are not tainted. Your cluster must have more than three zones for this behavior to take effect.
+
+You assign a node to a specific zone using the `topology.kubernetes.io/region` label in the node specification.
+
+.Sample node labels for Kubernetes zones
+[source,yaml]
+----
+kind: Node
+apiVersion: v1
+metadata:
+  labels:
+    topology.kubernetes.io/region=east
+----
+
+[id="nodes-edge-remote-workers-strategies-kubeconfig_{context}"]
+KubeConfig objects::
+--
+You can adjust the amount of time that the Kubernetes Controller Manager Operator (controller manager) checks the state of each node.
+
+This affects the timing of when the on-premise node controller marks nodes with the `Unhealthy` or `Unreachable` condition, create a KubeletConfig object with the `node-status-update-frequency` parameter:
+
+.Example KubeletConfig
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  name: disable-cpu-units
+spec:
+  machineConfigPoolSelector:
+    matchLabels:
+      machineconfiguration.openshift.io/role: worker
+  kubeletConfig:
+    node-status-update-frequency: <1>
+      - "5s"
+----
+<1> Specify the amount of time that the controller manager checks the state of each node associated with this MachineConfig. The default is `5s`.
+
+This parameter works with the `node-monitor-grace-period` and the `pod-eviction-timeout` parameters, which are not configurable. 
+
+* The `node-monitor-grace-period` parameter specifies how long the {product-title} waits after a node associated with this MachineConfig is marked `Unhealthy` if the controller manager cannot reach the node. Workloads on the node should continue to run after this time. If the remote worker node rejoins the cluster after the `node-monitor-grace-period` has expired, pods should continue to run. New pods can be scheduled to that node. The default is `40s`.
+
+* The `pod-eviction-timeout` parameter specifies the amount of time {product-title} waits after marking a node associated with this MachineConfig as `Unreachable` to start marking pods for eviction. Evicted pods are rescheduled on other nodes. If the remote worker node rejoins the cluster after `pod-eviction-timeout` has expired, the pods running on the remote worker node are terminated as the node controller has evicted the pods on-premise. Pods can then be rescheduled to that node. The default is `5m0s`.
```

Comment:
```suggestion
* The `pod-eviction-timeout` parameter specifies the amount of time {product-title} waits after marking a node that is associated with this MachineConfig as `Unreachable` to start marking pods for eviction. Evicted pods are rescheduled on other nodes. If the remote worker node rejoins the cluster after `pod-eviction-timeout` expires, the pods running on the remote worker node are terminated because the node controller has evicted the pods on-premise. Pods can then be rescheduled to that node. The `pod-eviction-timeout` period is `5m0s`.
```